src/android/CHIPTool/app/build.gradle

```
@@ -15,7 +17,7 @@ android {
 
         // NOTE: This build assumes CHIP was configured and built for armeabi-v7a. Deal with
         // other archs later when build is sane.
-        ndk {abiFilters 'armeabi-v7a'}
+//        ndk {abiFilters 'armeabi-v7a'}
```

nit: should we align the comment slashes here?
```suggestion
        // ndk {abiFilters 'armeabi-v7a'}
```

---

Done.